### ISSUE TITLE
boot2docker with macbook 2008 late is slow, what about docker remote?

### ISSUE THREAD

**Issue by minhoryang** [nanobox-io/nanobox#41]
boot2docker with macbook 2008 late is slow, what about docker remote?
It will be a prerequisite for supporting windows.

Thanks for reading this.

**Comment by tylerflint** [comment]
Thanks for the suggestion and insight. The help is greatly appreciated! 

Nanobox has been completely re-written from the ground up, and we just released the v1 into beta this week. While we still depend on boot2docker with this first release, the new structure opens up a world of possibilities to run nanobox on native or remote docker configurations. We would love to get your feedback on the new version. The new guides are still a WIP but should be usable within the next week. If you're interested, we'd love to talk with you on irc #nanobox or we can bring you into our slack channel as well.

Additionally, Nanobox and Nanobox Cloud have been merged into a unified process. If you would be interested in early access to the private beta please let me know and I can get you an access token.

Just reach out to hello@nanobox.io and we can get you going.

I really appreciate your help on this. I'll go ahead and close this ticket and hope to hear from you directly.
